tasks/verifySpokePool.ts

```diff
@@ -0,0 +1,73 @@
+import assert from "assert";
+import { task } from "hardhat/config";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { getDeployedAddress } from "../src/DeploymentUtils";
+import { SpokePool__factory } from "../typechain";
```

Comment:
This may be a problem; typechain _might_ not be available at the time this is built.

Reply:
Fixed: https://github.com/across-protocol/contracts/pull/582/commits/2059e18255be86314f32f267e203b204ca1a3b82